frontend/vite.config.ts

```diff
@@ -1,5 +1,6 @@
 import reactPlugin from '@vitejs/plugin-react';
 import { type ConfigEnv, defineConfig, loadEnv } from 'vite';
+import { VitePWA as VitePWAPlugin } from 'vite-plugin-pwa';
```

Comment:
Let's start it from the lowercase letter, because we have similar import above - reactPlugin
```suggestion
import { VitePWA as vitePWAPlugin } from 'vite-plugin-pwa';
```

Reply:
Oscar forget to change use of vitePwa to lower case in code

Reply:
Good point) will fix soon